CONTRIBUTING.md

```diff
@@ -446,9 +446,10 @@ Must be one of the following:
 -   **stark-core**
 -   **stark-demo**
 -   **stark-rbac**
--   **stark-starter**
 -   **stark-testing**
 -   **stark-ui**
+-   **showcase**
+-   **starter**
```

Comment:
Not sure whether we should change these scopes: we already have `stark-demo` (aka showcase) and `stark-starter` (aka starter) and we have used them frequently (check the [CHANGELOG](https://github.com/NationalBankBelgium/stark/blob/master/CHANGELOG.md)). The only one that we have rarely used and it was not defined is `showcase`.

I have no strong opinion on this, just the fact that we will have several names for these in our changelog. On the other hand, it is true that we haven't reached a final version so we can technically change the naming easily.

If we change the naming, then I agree that the `stark-` is not relevant and we should also remove `stark-demo` from the scopes too.

Reply:
As you say, we're not yet in final release. I think having a shorter name for those two should be better.
Good catch for the "stark-demo" ! 😊 (which is actually absent in "commitlint.config.js").

@nicanac What do you think ? Do you validate the following replacements ?
  - stark-demo ➡️ showcase
  - stark-starter ➡️ starter

Reply:
Yes sure, my only question is, are we going to rewrite the changelog ?

Reply:
Sorry, I didn't see your comment and I added mine... 
Actually, it concerns the starter and the showcase, it's linked to Stark itself. 
Because of that, I think it's not a big deal if the CHANGELOG has different values for "showcase", "stark-demo", "stark-starter" and "starter". 
Since the CHANGELOG is generated automatically based on the commit messages, we won't change it 😊